### PR TITLE
add movie service

### PR DESCRIPTION
**Pull request by HelenAndrade**
Created movie service and changed movie view to add service.

### CODE REVIEW

File: src/components/NavBar.vue
Comment (DarlanMendes) [inline src/components/NavBar.vue:1]: You must use english as default language coding. Substitute "Séries" for english word version.

Comment (HelenAndrade) [reply src/components/NavBar.vue:1]: Fixed path from Séries

File: src/views/Movies/MoviesView.vue
Comment (DarlanMendes) [inline src/views/Movies/MoviesView.vue:1]: Error isn't showed on this view. Errors are treated at axios config interceptors

Comment (HelenAndrade) [reply src/views/Movies/MoviesView.vue:1]: Deleted error and console.log's results

**Review by DarlanMendes — APPROVED** [review]

**Pull request merged**